test/controllers/Task.js

```diff
@@ -71,4 +72,39 @@ describe('Task', () => {
     });
   });
 
+
+  describe('#assign', () => {
+    it('add assign', done => {
+      let path = '/tasks/' + taskFixtures.task1._id.toString();
+      let data = {assigned: ['2', '1']};
```

Comment:
You should send ProfileId-s from fixtures. So import the Profile fixtures at the top and here use them. For example: let `data = {assigned: [profileFixtures.profile0._id.toString(), profileFixtures.profile...]};`